test/CurveHelper.ts

```diff
@@ -68,33 +71,52 @@ async function fundMetapool (addressPool, [amountLvUSD, amount3CRV], owner, r) {
     await token3CRV.approve(addressPool, amount3CRV);
     await lvUSD.approve(addressPool, amountLvUSD);
     const pool = await getMetapool(addressPool, owner);
-    const balanceLvUSD = await pool.balances(0);
-    const balance3CRV = await pool.balances(1);
-    // if the pool is NOT empty we calculated expected amount of minted LP
+    let balanceLvUSD = await pool.balances(0, {
+        gasLimit: 3000000,
```

Comment:
Is this needed? You added gas param in hardhat.config.ts

Reply:
For some reason the config param wasnt working so I had to do it here. I will try to run it again removed.